spec/.rubocop.yml

```diff
@@ -23,3 +23,6 @@ Style/SymbolProc:
 
 RSpec/FilePath:
   Enabled: false
+
+RSpec/ExampleLength:
+  Enabled: false
```

Comment:
This addition gets rid of the 5-line maximum for a spec example -- lmk if we want to do something different.

Reply:
Yeah I think this is fine.  in non rspec-land, there are some settings like

```ruby
Metrics/MethodLength:
  Max: 50
  CountAsOne: ['heredoc']
```

that would count your big heredoc string as one line, but I don't think the rspec cop has this feature